frontend/src/Components/Elements/Inputs/Input.js

```diff
@@ -51,6 +53,25 @@ function FJInput({
     return paddingProps;
   }
 
+  if (type === "textarea") {
+    return (
+        <Styled.Wrapper>
+          {label && (
+            <Input.Label errors={errors} py="2">
```

Comment:
The fields don't display errors for me:

<img width="1408" alt="image" src="https://user-images.githubusercontent.com/129679/168942092-5b20e8d8-8cc5-48a5-a0e1-39df0e5a843e.png">

Reply:
Nice catch, updated.